### PR TITLE
Update team group name

### PR DESCRIPTION
**Pull request by duanxianbo**
image

[(task:21559)](https://staging.teamyapp.com/teams/107/tasks/21559)

image

image

I got some issue to make the input to fit the content

### CODE REVIEW

**Comment by naturetime10** [comment]
Can we make text input background transparent and increase text input height? Can we use red `cross` mark instead of cross mark with circular background?

**Comment by naturetime10** [comment]
Can we update the screenshot? Thanks!

**Comment by duanxianbo** [comment]
> Can we update the screenshot? Thanks!

I am having some issue to make the input to fit the content size 
image

File: src/components/internal/Modals/TeamSettingsModal.component.tsx [hunk 1]
```diff
@@ -240,31 +252,100 @@ export class TeamSettingsModalComponent extends Component<Props, State> {
...
                         })}`}
                     >
-                        <div className={styles.Name}>{groupName}</div>
-                        {groupId && (
-                            <TooltipUI
-                                message='Delete'
-                                relativeLayout={this.props.deps.relativeLayout}
-                            >
+                        <div className={styles.NameInput}>
+                            {!isEditing ? (
```
Comment (naturetime10) [inline src/components/internal/Modals/TeamSettingsModal.component.tsx]: Can we change `!isEditing` to `isEditing` and swap the order of 
```
<div
                                    className={`${styles.Name} ${classNames({
                                        [styles.HasAction]: groupId,
                                    })}`}
                                    onDoubleClick={this.handleTeamMemberGroupNameClicked(
                                        groupName,
                                        groupId,
                                    )}
                                >
                                    {groupName}
                                </div>
```
and
```
<div
                                    className={`${styles.Name} ${classNames({
                                        [styles.HasAction]: groupId,
                                    })}`}
                                    onDoubleClick={this.handleTeamMemberGroupNameClicked(
                                        groupName,
                                        groupId,
                                    )}
                                >
                                    {groupName}
                                </div>
```

File: src/components/internal/Modals/TeamSettingsModal.component.tsx [hunk 2]
```diff
@@ -240,31 +252,100 @@ export class TeamSettingsModalComponent extends Component<Props, State> {
...
-                                    <MaterialIconUI>close</MaterialIconUI>
+                                    {groupName}
                                 </div>
-                            </TooltipUI>
-                        )}
+                            ) : (
+                                <div
+                                    className={styles.TextField}
+                                    contentEditable
+                                    onInput={this.handleTeamMemberGroupNameChanged(
```
Comment (naturetime10) [inline src/components/internal/Modals/TeamSettingsModal.component.tsx]:
```suggestion
                                    onInput={this.handleTeamMemberGroupNameInput(
```

File: src/components/internal/Modals/TeamSettingsModal.component.tsx [hunk 3]
```diff
@@ -603,6 +684,69 @@ export class TeamSettingsModalComponent extends Component<Props, State> {
         );
     }
 
+    private handleTeamMemberGroupNameClicked =
```
Comment (naturetime10) [inline src/components/internal/Modals/TeamSettingsModal.component.tsx]:
```suggestion
    private handleOnTeamMemberGroupNameClick =
```

File: src/components/internal/Modals/TeamSettingsModal.component.tsx [hunk 4]
```diff
@@ -603,6 +684,69 @@ export class TeamSettingsModalComponent extends Component<Props, State> {
...
+                    name,
+                },
+            };
+
+            this.setState({
+                editingTeamMemberGroupInput,
+            });
+        };
+
+    private handleTeamMemberGroupNameChanged =
```
Comment (naturetime10) [inline src/components/internal/Modals/TeamSettingsModal.component.tsx]:
```suggestion
    private handleTeamMemberGroupNameInput =
```

File: src/components/internal/Modals/TeamSettingsModal.component.tsx [hunk 5]
```diff
@@ -603,6 +684,69 @@ export class TeamSettingsModalComponent extends Component<Props, State> {
...
+        (groupId: number) => (event: ChangeEvent<HTMLDivElement>) => {
```
Comment (naturetime10) [inline src/components/internal/Modals/TeamSettingsModal.component.tsx]: Can we update the data type for `event`?

**Review by naturetime10 — CHANGES_REQUESTED** [review]

**Review by naturetime10 — APPROVED** [review]
Approved with a few clean up comments

**Pull request merged**